evm_loader/program/Cargo.toml

```diff
@@ -51,16 +51,14 @@ rlp = "0.5"
 static_assertions = "1"
 borsh = "0.9"
 bincode = "1"
-serde_bytes = "0.11"
-serde = { version = "1", features = ["derive"] }
+serde_bytes = "0.11.12"
+serde = { version = "1.0.183", default-features = false, features = ["derive", "rc"] }
```

Comment:
This `serde` version needs a bump too.

Reply:
Thanks, I've updated it everywhere now. :)